app/controllers/users/devices_controller.rb

```diff
@@ -125,4 +116,13 @@ def redirect_using_param_or_default(default: user_device_path(current_user.id, @
         redirect_to params[:redirect]
       end
     end
+
+    def invalid_uuid(msg)
```

Comment:
This is a nice idea! But why have it limited to just this error?
I'll merge this into dev, but tomorrow, have a go at putting this in the application controller and decoupling it, so that you can pass it an error message in any situation.

(On a side note, this sounds very Rails-y, might want to check if there's a method out there that already does this for you)